src/components/home/search/BookResults.tsx

```
@@ -1,53 +1,139 @@
+    };
+  });
+
+
+  const hasAnyFilter = Object.values(selectedFilters).some(
+    (v) => v && v.length > 0
+  );
+
+  const handleResetFilters = () => {
+    setSearchParams((prev) => {
+      const next = new URLSearchParams(prev);
+      FILTER_KEYS.forEach((key) => next.delete(key));
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)